src/Compilers/CSharp/Test/Emit/PDB/PDBIteratorTests.cs

```diff
@@ -190,6 +198,7 @@ System.Collections.Generic.IEnumerable<int> IEI<T>(int i0, int i1)
           <namespace usingCount=""0"" />
         </using>
         <hoistedLocalScopes>
+          <slot />
           <slot startOffset=""0x2a"" endOffset=""0xb4"" />
           <slot startOffset=""0x6e"" endOffset=""0xb2"" />
```

Comment:
I guess it's not but it would be good to find out what's going on

---
In reply to: [151315581](https://github.com/dotnet/roslyn/pull/22985#discussion_r151315581) [](ancestors = 151315581)

Reply:
(could be a follow up)

---
In reply to: [151507442](https://github.com/dotnet/roslyn/pull/22985#discussion_r151507442) [](ancestors = 151507442,151315581)